bring-it-up/bring-it-up
# Update data

## Description
#### Summary:
Updates data

#### Changes:
- Updates fields for some services in counselling-services.json
- Makes the 'hours' field optional
- Remove the 'isAllDay' field

#### How to test this PR:
- Check that there are no mistakes/typos in the data
- Run npm test and check that everything is still passing
  
## Checklist

- [x] I ran all the tests and they were successful
